### PR TITLE
update development version

### PR DESCRIPTION
**Pull request by lamhuy**
ran mvn release:update-versions -DdevelopmentVersion=1.6.0-SNAPSHOT

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.codice</groupId>
     <artifactId>alliance</artifactId>
-    <version>1.5.0-SNAPSHOT</version>
+    <version>1.6.0-SNAPSHOT</version>
```
Comment (bdeining) [inline pom.xml]: should update the DDF version too

**Comment by bdeining** [comment]
build now

**Comment by Schachte** [comment]
build now (artifacts are deployed now)

**Comment by cxbot1** [comment]
Internal build has been scheduled, your results will be available at build completion.

**Comment by cxbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://jenkins.codice.org/job/Alliance-Jobs/job/pr/job/Linux/1121/
:white_check_mark: JOB SUCCESS

**Pull request merged**